load/LoadOrganizations.py

```diff
@@ -98,101 +95,182 @@ def hasDamIds(self, languageRecord, typeCode):
 #				resultSet.add(name)
 
 
+	def get_lpts_licensors(self, language_record, fileset_id, type_code):
+		licensors = []
+		if language_record.Has_Ambiguous_Agreement():
+			print("WARN ambiguos agreement and it needs to be fixed for fileset id: %s" % (fileset_id))
+			return None
+
+		if language_record.Has_Other_Agreement():
+			print("WARN licensor: %s and co licensor: %s cannot be processed due to 'other Agreement' for fileset id: %s" % (language_record.Licensor(), language_record.CoLicensor(), fileset_id))
+			return None
+
+		if language_record.Has_Lora_Agreement() and type_code == "audio":
+			self.add_licensor_if_not_empty(licensors, language_record.Licensor(), fileset_id, "licensor")
+			return licensors
+
+		if language_record.Has_Lora_Agreement() and type_code == "text":
+			self.add_licensor_if_not_empty(licensors, language_record.CoLicensor(), fileset_id, "co-licensor")
```

Comment:
don't call if coLicensor is empty

Reply:
@bradflood , It has been updated as you can see in the following link: https://github.com/faithcomesbyhearing/dbp-etl/pull/130/files#diff-0628aedffa821d1b9acf4c81d50c756fe72674fcc7ca34c47e0a5b0a65c60aceR117